asv_bench/benchmarks/algos/isin.py

```diff
@@ -274,7 +271,15 @@ def time_isin(self, series_type, vals_type):
 
 class IsInLongSeriesLookUpDominates:
     params = [
-        ["int64", "int32", "float64", "float32", "object", "Int64", "Float64"],
+        [
+            "int64",
+            "int32",
+            "float64",
+            "float32",
+            "object",
+            Int64Dtype(),
```

Comment:
this is very odd, why doesn't `Int64` not work here?

Reply:
This was so the `type` attribute could be accessed to get the underlying `numpy` compatible type. I'll try to reorganize to avoid this